src/main/java/org/jfree/chart/date/SpreadsheetDate.java

```diff
@@ -42,6 +42,8 @@
  * Represents a date using an integer, in a similar fashion to the
  * implementation in Microsoft Excel.  The range of dates supported is
  * 1-Jan-1900 to 31-Dec-9999.
+ * (This year limit is removed to support years
+ * earlier than 1990, which is currently supported in Microsoft Excel.)
```

Comment:
Suggestion: Just remove the last sentence (starting on line 43 and ending on 44) - then no need for adding lines 45 and 46.